keyboards/3.0/xct-Tibt-t-k0-qwerty.xml

```diff
@@ -0,0 +1,494 @@
+      <row keys="q w e r t y u i o p open-square close-square pipe"/>
+      <row keys="a s d f g h j k l gtertsheg apos"/>
+      <row keys="z x c v b n m comma period cycle"/>
+      <row keys="tsheg"/>
+    </layer>
+    <layer modifiers="shift">
+      <row keys="tilde bang at hash dollar percent caret amp asterisk open-paren close-paren underscore plus"/>
+      <row keys="Q W E R T Y U I O P open-curly close-curly pipe"/>
+      <row keys="A S D F G H J K L colon double-quote"/>
+      <row keys="Z X C V B N M open-angle close-angle question"/>
+      <row keys="space"/>
+    </layer>
+    <layer modifiers="caps">
+      <row keys="yigmgo dzudrtags cap_2 cap_3 cap_4 cap_5 cap_6 cap_7 cap_8 9 0 chadrtags lhagrtags"/>
+      <row keys="Q W E R T Y U I O P open-square close-square backslash"/>
+      <row keys="A S D F G H J K L semi-colon apos"/>
+      <row keys="Z X C V B N M comma period slash"/>
+      <row keys="space"/>
+    </layer>
+    <layer modifiers="shift caps">
+      <row keys="tilde bang at hash dollar percent caret amp asterisk open-paren close-paren underscore plus"/>
+      <row keys="q w e r t y u i o p open-curly close-curly pipe"/>
+      <row keys="a s d f g h j k l colon double-quote"/>
+      <row keys="z x c v b n m open-angle close-angle question"/>
+      <row keys="space"/>
+    </layer>
+    <layer modifiers="altR">
+      <row keys="alt_grave alt_1 alt_2 alt_3 alt_4 alt_5 alt_6 alt_7 alt_8 alt_9 alt_0 hyphen equal"/>
+      <row keys="q w alt_e alt_r alt_t alt_y alt_u alt_i alt_o p open-square close-square tshegshad"/>
+      <row keys="alt_a alt_s alt_d f alt_g alt_h j k alt_l semi-colon apos"/>
+      <row keys="z alt_x c v b alt_n alt_m comma period slash"/>
+      <row keys="nbtsheg"/>
+    </layer>
+    <layer modifiers="shift altR">
+      <row keys="alt_tilde bang charrtags alt_hash dollar percent caret amp asterisk alt_openp alt_closep underscore plus"/>
+      <row keys="Q W alt_E alt_R alt_T Y alt_U alt_I alt_O P open-curly close-curly pipe"/>
+      <row keys="alt_A S D F G H J K alt_L colon double-quote"/>
+      <row keys="Z X C V B N M open-angle close-angle question"/>
+      <row keys="space"/>
+    </layer>
+
+  </layers>
+
+  <variables>
+    <set id="consonants" value="\u{0F40} \u{0F41} \u{0F42} \u{0F44} \u{0F45} \u{0F46} \u{0F47} \u{0F49} \u{0F4A} \u{0F4B} \u{0F4C} \u{0F4E} \u{0F4F} \u{0F50} \u{0F51} \u{0F53} \u{0F54} \u{0F55} \u{0F56} \u{0F58} \u{0F59} \u{0F5A} \u{0F5B} \u{0F5D} \u{0F5E} \u{0F5F} \u{0F60} \u{0F61} \u{0F62} \u{0F63} \u{0F64} \u{0F65} \u{0F66} \u{0F67} \u{0F68} \u{0F6A} \u{0F90} \u{0F91} \u{0F92} \u{0F94} \u{0F95} \u{0F96} \u{0F97} \u{0F99} \u{0F9A} \u{0F9B} \u{0F9C} \u{0F9E} \u{0F9F} \u{0FA0} \u{0FA1} \u{0FA3} \u{0FA4} \u{0FA5} \u{0FA6} \u{0FA8} \u{0FA9} \u{0FAA} \u{0FAB} \u{0FAD} \u{0FAE} \u{0FAF} \u{0FB0} \u{0FB1} \u{0FB2} \u{0FB3} \u{0FB4} \u{0FB5} \u{0FB6} \u{0FB7} \u{0FB8} \u{0FBA} \u{0FBB} \u{0FBC} \u{0FBE} " />
```

Comment:
When looking at this set of characters, it's very easy to miss that there are gaps in the range. Could we use a uset to shorten this and make the gaps more obvious? 

For example, this incomplete example (need to fill in `...`):

```
<uset id="consonants" value="[\u{0F40}-\u{0F42} \u{0F44}-\u{0F4C} ... \u{0FBA}-\u{0FBC} \u{0FBE}]" />
```